Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/Oni.yml

```
@@ -48,6 +48,9 @@
     - Nekomimetic
   - type: PotentiaModifier
     potentiaMultiplier: 0.5
+  - type: Tag
+    tags:
+    - Oni
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)